src/rental.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'book'
+require_relative 'person'
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] Remember that the rental is supposed to be adding itself to the @book achieving the 'belongs-to' association `book.rentals << self`, also this applies to the @person variable.